requirements.pinned.txt

```diff
@@ -6,190 +6,198 @@
-flask-oauthlib==0.9.4     # via invenio-oauth2server, invenio-oauthclient
+flask-oauthlib==0.9.5     # via invenio-oauth2server, invenio-oauthclient
 flask-principal==0.4.0    # via flask-security, invenio-admin
-flask-restful==0.3.6      # via flask-iiif
+flask-restful==0.3.7      # via flask-iiif
 flask-security==3.0.0     # via invenio-accounts
 flask-sqlalchemy==2.3.2   # via flask-alembic, invenio-db
+flask-webpackext==1.0.0   # via invenio-assets, invenio-i18n, invenio-search-ui
 flask-wtf==0.14.2
 flask==1.0.2
 fs==0.5.4                 # via invenio-files-rest
-future==0.16.0            # via cookiecutter, invenio-accounts, invenio-oauth2server
-html5lib==1.0.1           # via bleach
-idna==2.6                 # via cryptography, requests, urllib3
-idutils==1.0.1
+ftfy==4.4.3               # via invenio-records-rest
+functools32==3.2.3.post2 ; python_version < '3.0'  # via invenio-jsonschemas, jsonschema
+future==0.17.1            # via cookiecutter, invenio-accounts, invenio-oauth2server
+futures==3.2.0 ; python_version < '3.0'           # via tornado
+html5lib==1.0.1           # via ftfy
+idna==2.7                 # via cryptography, invenio-search, requests, urllib3
+idutils==1.1.1
 infinity==1.4             # via intervals
 intervals==0.8.1          # via wtforms-components
-invenio-access==1.0.0
-invenio-accounts==1.0.0
-invenio-admin==1.0.0
-invenio-assets==1.0.0
-invenio-base==1.0.1
+invenio-access==1.1.0
+invenio-accounts==1.0.2
+invenio-admin==1.1.1
+invenio-assets==1.1.1
+invenio-base==1.0.2
 invenio-cache==1.0.0
-invenio-celery==1.0.0
+invenio-celery==1.0.1
 invenio-communities==1.0.0a19
-invenio-config==1.0.0
-invenio-db[postgresql,versioning]==1.0.0
-invenio-formatter[badges]==1.0.0
-invenio-i18n==1.0.0
-invenio-iiif==1.0.0a3
-invenio-indexer==1.0.0
+invenio-config==1.0.1
+invenio-db[postgresql,versioning]==1.0.2
+invenio-files-rest==1.0.0a23  # via invenio-communities, invenio-iiif, invenio-records-files
+invenio-formatter[badges]==1.0.1
+invenio-i18n==1.1.1
+invenio-iiif==1.0.0a4
+invenio-indexer==1.0.1
 invenio-jsonschemas==1.0.0
-invenio-logging==1.0.0
-invenio-mail==1.0.1
-invenio-oaiserver==1.0.0
-invenio-oauth2server==1.0.0
-invenio-oauthclient==1.0.0
-invenio-opendefinition==1.0.0a7
+invenio-logging==1.1.0
+invenio-mail==1.0.2
+invenio-oaiserver==1.0.1
+invenio-oauth2server==1.0.2
+invenio-oauthclient==1.1.0
+invenio-opendefinition==1.0.0a8
 invenio-pages==1.0.0a4
 invenio-pidstore==1.0.0
 invenio-previewer==1.0.0a11
 invenio-query-parser==0.6.0  # via cds-dojson
-invenio-records-files==1.0.0a10
-invenio-records-rest==1.1.0
-invenio-records-ui==1.0.0
-invenio-records[postgresql]==1.0.0
-invenio-rest==1.0.0
-invenio-search-ui==1.0.1
-invenio-search[elasticsearch2]==1.0.0
+invenio-records-files==1.0.0a11
+invenio-records-rest==1.3.0
+invenio-records-ui==1.0.1
+invenio-records[postgresql]==1.0.1
+invenio-rest[cors]==1.0.0
```

Comment:
I am bit scared that this `[cors]` will create issue... did you add it because you have noticed a problem?